specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -3719,6 +5239,31 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
+    "EndpointTypeParameter": {
```

Comment:
>EndpointTypeParameter [](http://example.com/codeflow?start=5&length=21)

Recommend string enum if there are a limited set of values.

Reply:
actually can't do that same as type, the value will be Azure.OpenAI which can't be consider as enum

Reply:
See earlier comment.